### PR TITLE
#135842541 Game UI Revamp

### PR DESCRIPTION
**Pull request by jackiewmacharia**
#### What does this PR do?
 Changes the gaming user interface

 #### Description of Task to be completed?
 A user, after opting to start a new game, should see a redesigned interface of the game

 #### How should this be manually tested?
 Log in or play the game as guest

 #### What are the relevant pivotal tracker stories?
 #135842541

 #### Screenshots

_**Share game link page**_

share-link-page

_**Waiting for players to join page**_

waiting-for-players-to-join-game

_**Main game page**_

main-game-page

_**Czar page**_

czar-page

_**Czar choosing page**_

czar-choosing

_**Game ends page**_

game-ends

### CODE REVIEW

**Comment by gngchrs** [comment]
@andela-jmacharia Conflicts.

@andela-Bgathu @andela-cwekesa review

**Comment by jackiewmacharia** [comment]
@gangachris on it.

**Review by boswellgathu — APPROVED** [review]

**Comment by jackiewmacharia** [comment]
@gangachris merge conflicts solved, reviews currently ongoing.

File: public/views/answers.html [hunk 1]
```diff
@@ -27,25 +27,24 @@
...
-      <li>Each player begins with, and will always have, 10 white answer cards.</li>
-      <li>For each round, one player is randomly chosen as the Card Czar.</li>
-      <li>Everyone else answers the black question card by clicking on the answer card they want to use.</li>
-      <li>The Card Czar then picks a favorite answer, and whoever played that answer wins the round.</li>
-      <li>*Want to adorn your avatar with a glorious crown? Donate to charity after the game!</li>
-    </ol>
+    <div id="lobby-how-to-play" class="col-md-offset-3" style="color: #ffffff">
+      <h4>HANG ON</h4>
+      <br/>
+      <h6>We are finding you people to play with :)</h6>
```
Comment (jkanyua) [inline public/views/answers.html]: Check this out: http://fontawesome.io/icon/smile-o/
For the smile
<i class="fa fa-smile-o" aria-hidden="true"></i>

**Review by jkanyua — APPROVED** [review]

File: public/views/answers.html (hunk 1 above)
Comment (jackiewmacharia) [reply public/views/answers.html]: @andela-jkanyua done.

Comment (jackiewmacharia) [reply public/views/answers.html]: I've broken something with this change. Currently fixing it.

Comment (jackiewmacharia) [reply public/views/answers.html]: All is well now!

**Comment by jackiewmacharia** [comment]
@gangachris All is well now.

**Pull request merged**